server/src/main/java/com/paypal/selion/grid/servlets/SauceServlet.java

```diff
@@ -105,6 +109,19 @@ public void registerVirtualSauceNodeToGrid(HttpServletRequest req, HttpServletRe
         }
     }
 
+    private String getRegistrationRequestEntity() throws FileNotFoundException {
+        // get the max concurrent vm's allowed for the account from sauce labs
+        int maxConcurrent = SauceLabsRestApiFactory.getMaxConcurrency();
```

Comment:
I need to handle the condition here where the sauce labs api was not reachable and `getMaxconcurrency` returned `-1`